docs/tutorials/drupal8.md

```diff
@@ -217,3 +217,48 @@ Read More
 *   [Accessing services (eg your database) from the host](http://docs.devwithlando.io/tutorials/frontend.html)
 *   [Importing SQL databases](http://docs.devwithlando.io/tutorials/db-import.html)
 *   [Exporting SQL databases](http://docs.devwithlando.io/tutorials/db-export.html)
+
+### Setting up PHPStorm for running tests
+We want to be able to run PHPUnit, Kernel and Functional Test with PHPStorm instead by Commandline so we can debug our 
+tests. We want to be able to do this by the FPM from our lando app. To do this we need to setup a Remote interpreter 
+and make out Test Framework us it.
+
+* *Currently we have issues running functional test through PHPStorm. The reason this doesn't work yet is
+ that PHPStorm spins up a container and *
```

Comment:
And the reason is...?

Reply:
trevor the vampire https://www.youtube.com/watch?v=mUj88_aPNkg